### ISSUE TITLE
Running snickers in a privileged container

### ISSUE THREAD

**Issue by Ramblurr** [michielbdejong/snickers-proxy#38]
In the interest of keeping the host system tidy, what do you think of making a container that runs snickers w/ access to the docker socket to perform its duties?

**Comment by michielbdejong** [comment]
Yes, good idea! Do you know the command for this, do you have it working? Then we can add a Dockerfile to this repo and just mount config.json and /etc/snitch from the host, that's much cleaner indeed.
